src/qtgui/plotter.cpp

```diff
@@ -1218,34 +1218,34 @@ void CPlotter::draw(bool newData)
     const qreal shadowOffset = metrics.height() / 20.0;
 
     // Scale plotter for graph height
-    const double panddBGainFactor = plotHeight / fabs(m_PandMaxdB - m_PandMindB);
+    const float panddBGainFactor = (float)plotHeight / fabsf(m_PandMaxdB - m_PandMindB);
     // Scale waterfall and histogram for colormap
-    const double wfdBGainFactor = 256.0 / fabs(m_WfMaxdB - m_WfMindB);
+    const float wfdBGainFactor = 256.0f / fabsf(m_WfMaxdB - m_WfMindB);
 
-    const double fftSize = m_fftDataSize;
-    const double sampleFreq = m_SampleFreq;
-    const double fftCenter = (double)m_FftCenter;
-    const double span = (double)m_Span;
-    const double startFreq = fftCenter - span / 2.0;
-    const double binsPerHz = fftSize / sampleFreq;
+    const float fftSize = m_fftDataSize;
+    const float sampleFreq = m_SampleFreq;
+    const float fftCenter = (float)m_FftCenter;
+    const float span = (float)m_Span;
+    const float startFreq = fftCenter - span / 2.0f;
```

Comment:
Frequencies are one place where we may well need double precision, so I'll see about putting this back.

Reply:
Or maybe not. These frequencies are relative to the FFT center, so they might not have the same precision requirement that an absolute frequency would.

Reply:
I did end up switching frequency calculations back to double, because quantization was beginning to become visible far from the center frequency rate when using high sample rates. (For example, frequencies were quantized to 2 Hz when more than 16 MHz above or below the center frequency.)